evalml/tests/component_tests/test_feature_selectors.py

```diff
@@ -93,16 +93,17 @@ def test_feature_selectors_woodwork_custom_overrides_returned_by_components(X_df
     override_types = [Integer, Double, Boolean]
     for logical_type in override_types:
         try:
-            X = ww.DataTable(X_df, logical_types={0: logical_type})
-        except TypeError:
+            X = X_df
```

Comment:
Should this be `X = X_df.copy()` ? Same question for other usages I see in this PR.

Reply:
Yea I'll make that change. Thank you!